tdrs-frontend/src/components/GovBanner/GovBanner.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import GovBanner from './GovBanner'
+
+describe('GovBanner', () => {
+  it('should render a banner with government information', () => {
+    const wrapper = shallow(<GovBanner />)
+
+    expect(wrapper.find('.usa-banner')).toExist()
+  })
```

Comment:
It seems like it would be worthwhile to have a test that the banner expands.

Reply:
This is a similar problem to the Header.test.js issue where the only way to test is with class names and class names aren't updating in the tests.